vendor/engines/split_accounts/app/models/split_accounts/reports/export_raw_transformer.rb

```diff
@@ -12,7 +12,9 @@ def transform(original_hash)
       private
 
       def split_percent(order_detail)
-        number_with_precision(order_detail.split.percent, strip_insignificant_zeros: true) + "%" if order_detail.split
+        if order_detail.respond_to?(:split) && order_detail.split
+          number_with_precision(order_detail.split.percent, strip_insignificant_zeros: true) + "%"
+        end
       end
     end
   end
```

Comment:
I don't think I'm a fan either. The other option would be to decorate everything that goes through the transformer, even if it's just a single item. I think this is simpler.